sdk/containerregistry/azure-containerregistry/azure/containerregistry/_helpers.py

```diff
@@ -7,10 +7,17 @@
 import json
 import re
 import time
-from typing import Dict, List
+from typing import TYPE_CHECKING, List, Dict
```

Comment:
nit: List and Dict seem to only be used when type checking

Reply:
Correct. Typing imports don't need to be hidden behind a `if TYPE_CHECKING` statement. I don't know the exact specifics but pylint won't throw an error if it's in a type-hint somewhere in a page.